app/src/main/java/com/example/sopt_seminar/ui/ProfileFragment.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.sopt_seminar.ui
+
+import android.os.Bundle
+import android.view.View
+import com.example.sopt_seminar.R
+import com.example.sopt_seminar.databinding.ProfileFragmentBinding
+import com.example.sopt_seminar.util.BaseFragment
+
+class ProfileFragment : BaseFragment<ProfileFragmentBinding>(R.layout.profile_fragment) {
+    private val followerFragment = FollowerListFragment()
+    private val repoFragment = RepoFragment()
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        childFragmentManager.beginTransaction().replace(R.id.home_fragment_cv, followerFragment)
+            .commit()
+        binding.followerBtn.isSelected = true
+
+        binding.apply {
+            repo = repoFragment
+            follower = followerFragment
+            fragmentManager = childFragmentManager
+            main = this@ProfileFragment
+            drawable = R.drawable.kang
+        }
+    }
+
+    val changeState = fun(view: View) {
+        val followerBtn = binding.followerBtn
+        val repoBtn = binding.repoBtn
+        when (view.id) {
+            R.id.follower_btn -> {
+                if (!followerBtn.isSelected) {
+                    followerBtn.isSelected = true
+                    repoBtn.isSelected = false
+                }
+            }
+            R.id.repo_btn -> {
+                if (!repoBtn.isSelected) {
+                    repoBtn.isSelected = true
+                    followerBtn.isSelected = false
+                }
+            }
+        }
+    }
```

Comment:
변수를 함수화 시킬때 이것 또한 람다형으로 간결하게 표현 가능할듯
```suggestion
    val changeState : ((View) -> Unit) =  { view ->
        val followerBtn = binding.followerBtn
        val repoBtn = binding.repoBtn
        when (view.id) {
            R.id.follower_btn -> {
                if (!followerBtn.isSelected) {
                    followerBtn.isSelected = true
                    repoBtn.isSelected = false
                }
            }
            R.id.repo_btn -> {
                if (!repoBtn.isSelected) {
                    repoBtn.isSelected = true
                    followerBtn.isSelected = false
                }
            }
        }
    }
```

Reply:
해당 부분은 없어졌지만 다음에 참고하겠습니다!!!!